Ansible/roles/nginx/vars/main.yml

```diff
@@ -0,0 +1 @@
+nginx_conf_path: "/etc/nginx/sites-available/default"
```

Comment:
make sure to create custom configuration file name instead of using default as configuration file.